Upstream intends to include qt/aqt/data in the wheel under _aqt/ but
ends up including it under aqt/ as well. This is a minimal version of
the patch I sent upstream:
https://github.com/ankitects/anki/pull/4180
From: Lucio Sauer <watermanpaint@posteo.net>
--- a/qt/pyproject.toml
+++ b/qt/pyproject.toml
@@ -77,7 +77,7 @@ ankiw = "aqt:run"
 
 [tool.hatch.build.targets.wheel]
 packages = ["aqt"]
-exclude = ["**/*.scss", "**/*.ui"]
+exclude = ["aqt/data", "**/*.ui"]
 
 [tool.hatch.version]
 source = "code"
-- 
2.49.0

